openhtf/core/phase_executor.py

```diff
@@ -199,9 +199,10 @@ def execute_phase(self, phase):
     while not self._stopping.is_set():
       outcome = self._execute_phase_once(phase)
```

Comment:
I think line 217 should return `PhaseOutcome(openhtf.PhaseResult.CONTINUE)` instead, returning None there breaks the calling convention of `_execute_phase_once()`.